### PR TITLE
Add opam file for HoTT library for Coq 8.12

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

File: released/packages/coq-hott/coq-hott.8.12/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+To use the HoTT library, the following flags must be passed to coqc:
+   -noinit -indices-matter
+To use the HoTT library in a project, add the following to _CoqProject:
+   -arg -noinit
+   -arg -indices-matter
+"""
+tags: [ "logpath:HoTT" ]
+url {
+  src: "https://github.com/HoTT/HoTT/archive/refs/tags/V8.12.tar.gz"
+  checksum: "sha512=dfdc9360da69ff4aa78b425c3393de4fc824f6cdb522d7c147dac8e8f14ebda2882170aa3eae28d8c60135e44610cddc135e6d9093a7d6da06b8cd0395d8a812"
```
Comment (palmskog) [inline released/packages/coq-hott/coq-hott.8.12/opam:28]:
```suggestion
  checksum: "sha512=0c5b797373a409ed54ec8b80db5cbc3173a7898e983f8070a8c83df03368c59d1f8f6338ada0d5cf1d8385307060134742d0a76d91bd88822ceda409aec3f1b6"
```

**Comment by Alizter** [comment]
@palmskog Thanks!

**Comment by palmskog** [comment]
@Alizter this archive doesn't have a `Makefile`. Looks like it uses `autoconf`. You may want to add the `autoconf` command/dependency (or manually create an archive where `autoconf` has generated the required `Makefile`).

**Comment by Alizter** [comment]
@palmskog I've copied the config from the 8.11 version. It should be the same.

**Pull request merged**